src/Process/PlanarTileProcessing.js

```diff
@@ -24,17 +24,6 @@ function _isTileBigOnScreen(camera, node) {
     return (dim.x >= 256 && dim.y >= 256);
 }
 
-export function prePlanarUpdate(context, layer) {
```

Comment:
same part that [here](https://github.com/iTowns/itowns/pull/815/files#diff-989ef5eaaef422d31895b5f0e7eb2c7aL46)
There's the same `preUpdate` in `globePreUpdate` ?
It's used for elevation with color data , can use color data for elevation in the `globeLayer`?

Reply:
It is the same indeed, so I moved it to TiledGeometryLayer.